manifests/w/WiresharkFoundation/Wireshark/3.6.5/WiresharkFoundation.Wireshark.installer.yaml

```diff
@@ -0,0 +1,36 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: WiresharkFoundation.Wireshark
+PackageVersion: 3.6.5
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://www.wireshark.org/download/win64/Wireshark-win64-latest.msi
+  InstallerSha256: A7F3D8B733C97B7533C4AAC54379C82F1049B49C5F30D7E6CC29D7281976FA32
+  ProductCode: '{7A63969B-4046-486B-A6ED-2978EA327E18}'
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://www.wireshark.org/download/win64/Wireshark-win64-latest.exe

```

Comment:
```suggestion
  InstallerUrl: https://www.wireshark.org/download/win64/all-versions/Wireshark-win64-3.6.5.exe
```

Reply:
Using versioned URL is better here instead of pointing to the latest release.

It allows historic releases to be kept in this repository and prevents installation breakage due to the hash changing every Wireshark release.